src/lowerCaseEmails.js

```diff
@@ -0,0 +1,93 @@
+const dotenv = require("dotenv");
+
+dotenv.config({path: "../.env" ,silent: true});
+
+const mongoose = require("mongoose");
+
+const {MONGO_URI} = process.env;
+
+const options = {
+  useNewUrlParser: true,
+  useCreateIndex: true,
+  dbName: "shellhacks",
+  useFindAndModify: false
+};
+
+const db = () => Promise.resolve(mongoose.connect(MONGO_URI, options));
```

Comment:
an instance of the db function is already being called in another file, no need to re run it again